src/resources/org/firebirdsql/firebird_0_error_msg.properties

```diff
@@ -398,7 +398,7 @@
 335544714=invalid blob id
 335544715=Operation not supported for EXTERNAL FILE table {0}
 335544716=Service is currently busy\: {0}
-335544717=stack size insufficent to execute current request
+335544717=stack size insufficient to execute current request
```

Comment:
This is sourced from the Firebird code, so the typo would need to be fixed there (the next time I refresh the error codes will undo this change).